components/Article/PayNote.js

```diff
@@ -255,35 +282,33 @@ export const PayNote = compose(
     documentId,
     repoId,
     series,
-    position
+    position,
+    customPayNotes
   }) => {
-    const variation = getPayNoteVariation({
+    const payNote = getPayNote({
       inNativeIOSApp,
       isEligibleForTrial,
       series,
       trial,
       seed,
-      query
+      query,
+      customPayNotes
     })
     const lead = (
-      <Translation
-        baseKey={variation.key}
-        position={position}
-        element='title'
-      />
+      <Translation payNote={payNote} position={position} element='title' />
     )
-    const body = <Translation baseKey={variation.key} position={position} />
+    const body = <Translation payNote={payNote} position={position} />
     const payload = {
       documentId,
       repoId,
-      variation: variation.keyShort,
+      payNote: payNote.keyShort,
```

Comment:
```suggestion
      variation: payNote.keyShort,
```
Probably not renamed on purpose? This is sent to the backend for analysis.